charts/rancher-vsphere-csi/Chart.yaml

```diff
@@ -21,4 +21,4 @@ maintainers:
 name: rancher-vsphere-csi
 sources:
 - https://github.com/kubernetes-sigs/vsphere-csi-driver
-version: 3.3.1-rancher1
+version: 3.1.2-rancher3
```

Comment:
why is the version going backwards? that doesn't seem right.

Reply:
Ok, so which version I should add?

Reply:
@jiaqiluo was asking you to change only the `-rancherX` portion, since all you're changing is the chart. So now that you've rebased, just make it `3.3.1-rancher2`